dashboard/legacy/middleware/helpers/asset_bucket.rb

```diff
@@ -43,12 +43,14 @@ def copy_level_starter_assets(src_channel, dest_channel)
     channel = ChannelToken.find_by(storage_id: src_owner_id, storage_app_id: src_storage_app_id)
     return unless channel
 
+    # For a template-backed level, the level associated with the channel
+    # is the template level
     level = Level.cache_find(channel.level_id)
-    return unless level
+    return unless level&.starter_assets
 
     dest_owner_id, dest_storage_app_id = storage_decrypt_channel_id(dest_channel)
 
-    (level&.project_template_level&.starter_assets || level.starter_assets || []).map do |friendly_name, uuid_name|
+    level.starter_assets.map do |friendly_name, uuid_name|
```

Comment:
can you add a comment here explaining why we can always just copy the assets on the level?

Reply:
updated, LMK what you think!

Reply:
Thanks for adding the clarifying comment! [This file](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/parent_levels_child_level.rb#L68) helped me understand the relationship between 'derived' or 'child' level and the project_template_level.